src/selectors/getChildResolvedPath.ts

```diff
@@ -0,0 +1,24 @@
+import { SimplePath, Path } from '../types'
+import { parentOf, head, pathToContext, unroot } from '../util'
+import { State } from '../util/initialState'
+import isContextViewActive from './isContextViewActive'
+
+/** Get thoughtsResolved for a child from it's parent thoughtsResolved.
```

Comment:
Better description needed, e.g. Appends the head of a child `SimplePath` to a parent `Path`.